how-tos/run-function-on-startup.md

```diff
@@ -0,0 +1,30 @@
+# How to Run plugin code on app startup
+Copy the following into a startup.js file in the plugin folder:
```

Comment:
We should also mention about pre-startup, post-startup phases here.

Reply:
Is there an after startup phase? I am familiar with `preStartup` but what's the other one? I don't see anything about that in `api-core`?

Reply:
No there isn't just the preStartup.

Reply:
But I still think we should mention about `preStartup`, `startup` & `shutdown` here. This seems out of scope for the title of the doc. So maybe this could be a "run function lifecycle" doc?

Reply:
I think mentioning only startup in a doc entitled "running things on startup" makes a lot of sense